watershed-fire/2_process.R

```diff
@@ -0,0 +1,30 @@
+source("2_process/src/chart_data.R")
+source("2_process/src/map_data.R")
+
+# Prepare data for map and chart animations
+p2_targets <- list(
+  
+  # Define years as target (so they can be mapped for the map png branching)
+  tar_target(Years,
+             seq(years[1], years[2])),
+  
+  tar_target(Years_0.5,
+             seq(years[1], years[2], by = 0.5)),
```

Comment:
I like what you're doing here. I would suggest a slight modification to the naming of `Years` and `years` to avoid confusion, possibly changing `years` to `year_range` or something like that. Instead of using the indexing, you could also use `min(year_range)` and `max(year_range)` to make it a bit more flexible.